app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.spec.tsx

```diff
@@ -6,8 +6,8 @@ import { rest } from 'msw';
 import type { SetupServer } from 'msw/node';
 import { SWRConfig } from 'swr';
 import { I18nContext } from '@18f/identity-react-i18n';
+import { ADDRESS_SEARCH_URL, LOCATIONS_URL } from '@18f/identity-address-search';
 import { ComponentType } from 'react';
-import { ADDRESS_SEARCH_URL, LOCATIONS_URL } from './address-search';
```

Comment:
Is the original `address-search.tsx` meant to be removed with this work?

Reply:
Good question, i was thinking about that. I am working on another card, so this PR submitted for opinions.

Yes, i think we should remove the old one.